src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -528,6 +528,63 @@ bool IBinaryInteger<byte>.TryWriteLittleEndian(Span<byte> destination, out int b
         /// <inheritdoc cref="INumberBase{TSelf}.Abs(TSelf)" />
         static byte INumberBase<byte>.Abs(byte value) => value;
 
+        /// <inheritdoc cref="INumberBase{TSelf}.CreateChecked{TOther}(TOther)" />
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We don't have a language mechanism for explicitly invoking the base implementation?  That seems like a hole (we probably discussed) that we should address.  It's one thing to have a bit of boilerplate that forces us to redeclare the method as a one-liner that just delegates, but haven't to redo the whole implementation is unfortunate.

Reply:
Right, this is a general (and unfortunate) "issue" with DIMs. Ideally the language team comes up with a way to give access to this.